packages/components/src/presets/next/index.ts

```diff
@@ -20,6 +20,20 @@ const config: Config = {
       screens: {
         xl: "1240px",
       },
+      keyframes: {
+        buttonPulse: {
+          "0%, 100%": { backgroundColor: "rgba(0, 0, 0, 0.65)" },
+          "50%": { backgroundColor: "rgba(0, 0, 0, 0.325)" },
```

Comment:
pog

Reply:
[low impact, no actionable]  wondering if we care about their [motion preference](https://tailwindcss.com/docs/transition-property#supporting-reduced-motion) but given that our codebase as a whole doesn't really have this, i'm ok with skipping also (majority of the time this has no impact)

Reply:
nice catch, updated, thanks!